packages/block-library/src/gallery/test/index.native.js

```diff
@@ -622,6 +622,20 @@ describe( 'Gallery block', () => {
 		expect( getEditorHtml() ).toMatchSnapshot();
 	} );
 
+	it( 'does not display MediaReplaceFlow component within the block toolbar', async () => {
+		const screen = await initializeWithGalleryBlock( {
+			numberOfItems: 3,
+			media,
+		} );
+		const { queryByTestId } = screen;
```

Comment:
I [recommend](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-screen) avoiding destructuring the result of [`render`](https://callstack.github.io/react-native-testing-library/docs/api#render) (i.e. `initializeWithGalleryBlock`) as destructuring requires updating the destructured values whenever adding or changing queries. 

Unrelated to this specific change: when adding new test files or refactoring existing tests, I recommend leveraging the [`screen`](https://testing-library.com/docs/queries/about/#screen) import as well. There is little usage in the project so far. My practice is to only use it in new test files as I didn't want to create confusion by intermingling its usage in existing tests.

Reply:
> Unrelated to this specific change: when adding new test files or refactoring existing tests, I recommend leveraging the [`screen`](https://testing-library.com/docs/queries/about/#screen) import as well. There is little usage in the project so far. My practice is to only use it in new test files as I didn't want to create confusion by intermingling its usage in existing tests.

That's interesting. I saw it was introduced in [version `10.1.0` of the RN testing library](https://github.com/callstack/react-native-testing-library/releases/tag/v10.1.0), that explains why we didn't use it when we started to implement integration tests.

In relation to this, at some point, we should update the [integration test guide](https://github.com/WordPress/gutenberg/blob/trunk/docs/contributors/code/react-native/integration-test-guide.md#rendering) to reflect this.